### DISCUSSION TITLE
Dịch entailment

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#156] [Terminology] [answered]
Context: 1 task trong NLP
> Entailment (whether sentence A entails sentence B). For example, the sentence "Jane had a lot of fun at her friend’s birthday party" entails "Jane enjoyed the party," but it is contradicted by "Everyone hated the party" and it is unrelated to "The Earth is flat."

Từ này mình nghĩ có thể dịch là "suy luận", nhưng mà như vậy lại trùng với "inference". Mọi người có cao kiến gì không nhỉ?

@mlbvn/handson-ml

**Comment by ngcthuong** [comment]
Đồng nghĩa, tương tự, từ này không phải từ khoá chính dịnh du di cũng đc.

**Reply by minhduc0711** [reply]
Em hiểu "A entails B" là "từ A suy ra được B là đúng" nên em thấy "đồng nghĩa" chưa đúng lắm.

**Reply by hnguyentt** [reply]
Entailment: Hệ quả logic

Ref: https://en.wikipedia.org/wiki/Logical_consequence

**Reply by ngcthuong** [reply]
Một like cho *hệ quả*

**Comment by lkhphuc** [comment, marked as answer]
Đồng ý @nguyenhoa93, trước giờ mình cũng toàn hiểu là `hệ quả`. 
Như trong ngữ cảnh NLP ở trên thì A entails B dịch là `A có hệ quả là B` hay `B là hệ quả của A`
